webapp/src/Controller/API/JudgementController.php

```diff
@@ -32,7 +31,7 @@ class JudgementController extends AbstractRestController implements QueryObjectT
     /**
      * @var string[]
```

Comment:
This can be removed?

Reply:
How? We can’t put it in a php type

Reply:
see above but this one is indeed less clear.